doc/classes/Input.xml

```diff
@@ -293,6 +293,8 @@
 			</argument>
 			<description>
 				Set a custom mouse cursor image, which is only visible inside the game window. The hotspot can also be specified. Passing [code]null[/code] to the image parameter resets to the system cursor. See enum [code]CURSOR_*[/code] for the list of shapes.
+				[code]image[/code] must be less than 256x256.
+				[code]hotspot[/code] must be inside image.
```

Comment:
@akien-mga is this ok?

Reply:
I would say `[code]hotspot[/code] must be within [code]image[/code]'s size.`

Reply:
And `[image]'s size must be lower than 256x256.`

Reply:
Ok, changed.